### DISCUSSION TITLE
Resizing image

### DISCUSSION THREAD

**Discussion by Grazulex** [filamentphp/filament#3608] [Help]
Hello,

I have a upload image field in one of my form :
```

                Forms\Components\FileUpload::make('picture')
                    ->image()
                    ->imageCropAspectRatio('1:1')
                    ->imageResizeMode('contain')
                    ->imageResizeTargetWidth('1000')
                    ->imageResizeTargetHeight('490'),
```
The ratio is ok but nothing happend with the resizing....my picture is steal bigger than 1000*490
Do I need to add something more ?

Thanks you

**Comment by ryangjchandler** [comment]
@Grazulex Those options are used to change how the preview looks, not how the final uploaded image looks. For that, you'll need to use a server-side tool like VIPS or ImageMagick.
